lib/modules/dosomething/dosomething_reportback/dosomething_reportback.admin.inc

```diff
@@ -314,32 +314,31 @@ function theme_dosomething_reportback_files_form($variables) {
   $rows = array();
   foreach ($rb_files as $fid) {
     $rbf = reportback_file_load($fid);
+    $reportback = reportback_load($rbf->rbid);
```

Comment:
@aaronschachter oh shoot - that was leftover from adding `first_name` and `avatar` - thanks for catching that! Deleted :)